### ISSUE TITLE
Why gradient clipping = 12 ? 

### ISSUE THREAD

**Issue by SimJeg** [MIC-DKFZ/nnUNet#1224]
Hello,

I noticed that gradient clipping is used in `nnUNetTrainerV2.py`, with a hard-coded max norm of 12 : 

```python
torch.nn.utils.clip_grad_norm_(self.network.parameters(), 12)
```

Why 12 ? Smaller values (e.g. 1) are usually used in the litterature.

**Comment by function2-llx** [comment]
Hi @SimJeg , I'm also interested in this question. Have you tried smaller values and observed any difference?

**Comment by SimJeg** [comment]
@function2-llx I decrease it to 1 among other modifications, so don't have nice ablation study to share (training was a bit long for my use case)

**Comment by FabianIsensee** [comment]
A very long time ago I optimized this value and 12 was what I ended up with. Gradient clipping is only there to prevent (very rare) nan-related problems. I am not sure if we even still need this
